pages/linux/rig.md

```diff
@@ -0,0 +1,24 @@
+# rig
+
+> Utility that will piece together a random first name, last name, street number and address, along with a geographically consistant (ie, they all match the same area) city, state, ZIP code, and area code.
+> More information: <http://manpages.ubuntu.com/manpages/bionic/man6/rig.6.html>.
+
+- Return a random name and address:
+
+`rig`
+
+- Specify male/female name. If neither or both options are selected, RIG uses both female and male names:
+
+`rig -m`
+
+- Use data files found in *datadir*. Without this  option, the  default  directory  of /usr/share/rig is assumed:
+
+`rig -d {{datadir}}`
+
+- Output *num* identities. Default is 1:
```

Comment:
Do you think we need to explain default is 1?
```suggestion
- Display a specific `count` of identities:
```

Reply:
Nope I think we can leave that one out.